core/__tests__/tasks/appRefreshQuery/check.ts

```diff
@@ -76,19 +92,31 @@ describe("tasks/appRefreshQuery:check", () => {
       await anotherApp.update({ state: "deleted" });
 
       new AppRefreshQuery({
-        appId: app.id,
+        appId: anotherApp.id,
         refreshQuery: "SELECT 'hi' AS name;",
         state: "ready",
       });
       await appRefreshQuery.save();
 
-      await task.enqueue("appRefreshQuery:check", {}); //does not throw
+      await task.enqueue("appRefreshQuery:check", { appRefreshQuery }); //does not throw
 
       //also does not enqueue any runs
       const enqueuedRuns = await Run.findAll({
         where: { creatorId: schedule.id, state: "running" },
       });
       expect(enqueuedRuns.length).toBe(0);
     });
+
+    test("only schedules marked 'refreshEnabled:true' are triggered", async () => {
+      await schedule.update({ refreshEnabled: false });
+
+      await task.enqueue("appRefreshQuery:check", { appRefreshQuery }); //does not throw
+
+      const enqueuedRuns = await Run.findAll({
+        where: { creatorId: schedule.id, state: "running" },
+      });
+
+      expect(enqueuedRuns.length).toBe(0);
+    });
```

Comment:
There isn't a test that actually enqueues any runs.  Please add a test like this one, but that starts with `await schedule.update({ refreshEnabled: true });` and see that there are enqueued runs